es-ds-components/utils/mime-type-finder.ts

```diff
@@ -39,10 +39,10 @@ const mimeTypesAndHexes = [
 
 export const mimeTypes = mimeTypesAndHexes.map(({ mime }) => mime);
 
-export function getHumanReadableTypeFromMime(mimeType) {
+export function getHumanReadableTypeFromMime(mimeType: any) {
     return mimeTypesAndHexes.find((mimeTypeAndHex) => mimeTypeAndHex.mime === mimeType)?.humanReadable;
 }
 
-export default function findMimeType(hex) {
```

Comment:
Do you know if this is being imported anywhere using the default export?

Reply:
It's being used [in es-file-input](https://github.com/EnergySage/es-ds/blob/bb6141913fd3d399f795a4ee45d344c4f9ae6ecc/es-ds-components/components/es-file-input.vue#L109). Prior to this change it was manually imported in that file.